contracts/tests/TestContract.sol

```diff
@@ -18,7 +18,7 @@ contract TestContract is debtTrackingInterface {
     function increaseDebt(address creditor, uint64 value) external {
     }
 
-    function getDebt(address debtor, address creditor) external view returns (int256) {
+    function getDebt(address, address) external view returns (int256) {
```

Comment:
This makes the code harder to understand even though the two parameters are not being used. The same is true for the removed `bytes memory returnedData` above.

Reply:
removing `bytes memory returnData` is actually more gas efficient. I agree that removing the two parameters make it harder to understand but it's the only way to fix (or silence) the compiler warnings, and they are printed every time a test fails which is annoying.